src/session-search.ts

```
@@ -4,7 +4,402 @@
+// ---------------------------------------------------------------------------
+// Query language
+// ---------------------------------------------------------------------------
+
+/** Fields a term can be aimed at with `field:value`. */
+export type ScopedField =
+  | 'title'
+  | 'branch'
+  | 'msg'
+  | 'project'
+  | 'account'
+  | 'recap';
```

ESLint

[error] Replace `|·'title'⏎··|·'branch'⏎··|·'msg'⏎··|·'project'⏎··|·'account'⏎·` with `'title'·|·'branch'·|·'msg'·|·'project'·|·'account'`

(prettier/prettier)